sotodlib/site_pipeline/preprocess_tod.py

```diff
@@ -118,7 +118,9 @@ def preprocess_tod(
         logger.info(f"Beginning run for {obs_id}:{group}")
 
         aman = context.get_obs(obs_id, dets={group_by:group})
-        proc_aman = pipe.run(aman)
+        proc_aman, success = pipe.run(aman)
+        if success != 'end':
```

Comment:
@kmharrington I added this here because if it tried to save the `proc_aman` to h5 for a group that failed then the loading failed because some of the subfields under the preprocess axis manager were missing for that group (wafer slot). This instead just doesn't save that wafer to h5 at all then if you use `on_missing: trim` for the preprocess section in your context it loads fine just skipping that wafer in the load.